src/rust/rust-proto/src/graplinc/grapl/api/plugin_sdk/analyzers/v1beta1/client.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
?

Reply:
unless we're saying there will be no rust clients, to which i'd say, pls leave a comment

Reply:
whoops